sdk/golang/ccnp/quote/proto/quote-server.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+package quoteserver;
+option go_package = "github.com/intel/confidential-cloud-native-primitives/sdk/golang/ccnp/quote/proto/quoteServer";
+
+message HealthCheckRequest {
+  string service = 1;
+}
+
+message HealthCheckResponse {
+  enum ServingStatus {
+    UNKNOWN = 0;
+    SERVING = 1;
+    NOT_SERVING = 2;
+    SERVICE_UNKNOWN = 3;
+  }
+  ServingStatus status = 1;
+}
+
+service GetQuote {
+    rpc GetQuote (GetQuoteRequest) returns (GetQuoteResponse);
+}
+
+message GetQuoteRequest {
+   string user_data = 1;
+   string nonce = 2;
```

Comment:
What's the meaning of the word "nonce" here?

Reply:
A nonce is a number/string that is used only once in a cryptographic operation. It provides protection against a replay attack.